doc/releases/release-notes-3.6.rst

```diff
@@ -149,6 +149,17 @@ Boards & SoC Support
 Build system and infrastructure
 *******************************
 
+* Added functionality for Link Time Optimisation.
+  This change includes interrupt script generator rebuilding and adding following options:
+
+  - :kconfig:option:`CONFIG_ISR_TABLES_LOCAL_DECLARATION` Kconfig option:
+    LTO compatible interrupt tables parser,
+  - :kconfig:option:`CONFIG_LTO` Kconfig option: Enable Link Time Optimisation.
+
+  Currently the LTO compatible interrupt tables parser is only supported by ARM architectures and
+  GCC compiler/linker.
+  See `pull request 66392 <https://github.com/zephyrproject-rtos/zephyr/pull/66392>`_ for details.
```

Comment:
hardcoding links in the documentation should be avoided. 

```suggestion
  See pull request :github:`66392` for details.
```

Reply:
Done